src/applications/mhv/secure-messaging/containers/FolderThreadListView.jsx

```diff
@@ -179,15 +179,15 @@ const FolderThreadListView = props => {
     if (folder?.folderId !== null) {
       dispatch(
         getListOfThreads(
-          folder.folderId,
+          folder?.folderId,
```

Comment:
this optional chaining is not needed as we are already validating on line 179 in `if (folder?.folderId !== null) ` that the `folderId` is not null

Reply:
valid.